<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>100</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox5">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>100</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="selectButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>530</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>筛选</string>
    </property>
   </widget>
   <widget class="QPushButton" name="userpictrue">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>160</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="loginsetting">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>160</width>
      <height>160</height>
     </rect>
    </property>
    <property name="text">
     <string>登陆设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_dish">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>160</width>
      <height>160</height>
     </rect>
    </property>
    <property name="text">
     <string>添加菜品</string>
    </property>
   </widget>
   <widget class="QPushButton" name="myfriend">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>160</width>
      <height>160</height>
     </rect>
    </property>
    <property name="text">
     <string>我的好友</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>主食</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>辣度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>荤素</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>中/西餐</string>
    </property>
   </widget>
   <widget class="QLabel" name="label5">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>60</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>风味</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>781</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/new/prefix1/images/PKUmap.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>燕南食堂</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>学五食堂</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>家园食堂</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>430</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>学一食堂</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
